_maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm

```diff
@@ -37099,10 +37099,10 @@
 /area/ice_colony/underground/maintenance/east)
 "bTY" = (
 /obj/machinery/atmospherics/pipe/simple/green/hidden,
+/obj/effect/decal/remains/human,
 /obj/machinery/light/small{
-	dir = 8
+	dir = 4
```

Comment:
Is this change intended?

Reply:
Isn't that from PR that are already merged?  Spookydonut said there's nothing more to be done to this commit

Reply:
yeah this looks like an unintentional change